libfabric.spec

```diff
@@ -2,7 +2,6 @@
 %global major 1
 %global minor 15
 %global bugrelease 0
-%global prerelease rc3
```

Comment:
Maybe just comment this out so that a future prerelease update doesn't have to figure out to add this line back in?

Reply:
good point. fixed